Tests/DatadogTests/Datadog/Tracing/DDSpanTests.swift

```diff
@@ -46,6 +60,101 @@ class DDSpanTests: XCTestCase {
         XCTAssertEqual(span.ddContext.baggageItems.all, ["foo": "bar", "bizz": "buzz"])
     }
 
+    // MARK: - Errors
+
+    func testSettingErrorFromSwiftError() throws {
+        let span: DDSpan = .mockWith(operationName: "operation")
+        XCTAssertEqual(span.logFields.count, 0)
+
+        span.setError(ErrorMock())
+
+        XCTAssertEqual(span.logFields.count, 1)
+        let logFields = span.logFields.first!
+        XCTAssertNotEqual(logFields.count, 0)
+        try XCTAssertEqual(logFields.otEvent(), "error")
+        let spanErrorStack = try logFields.otStack()
+        let fileName = #fileID.components(separatedBy: "/").last!
+        XCTAssertTrue(spanErrorStack.contains(fileName))
```

Comment:
If we use `#sourceLocation`:
```swift
#sourceLocation(file: "/user/abc/Foo.swift", line: 100)
span.setError(ErrorMock())
#sourceLocation()
```
this will become:
```swift
try XCTAssertEqual(logFields.otStack(), "Foo.swift: 100")
```

Reply:
That's a great tip, thanks!